packages/portal-proto/src/components/BtnWithHoverCallout.tsx

```diff
@@ -1,6 +1,47 @@
 import tw from "tailwind-styled-components";
 
-const BtnWithHoverCallout = tw.button`
+const BtnWithHoverCalloutTopArrow = tw.button`
+  relative
+  flex items-center w-full text-left
+  hover:bg-primary-lightest
+  hover:after:content-['']
+  hover:after:absolute
+  hover:after:top-[-8px]
+  hover:after:left-1/2
+  hover:after:transform
+  hover:after:translate-x-[-50%]
+  hover:after:border-l-[8px]
+  hover:after:border-l-solid
```

Comment:
These border solid styles are overwritten on the next line by the border transparent (I see the original component had this mistake too which I think is my bad 🙀 ).

Reply:
I just updated. Yes I overlooked at it too. TBH I am still confused with that css 😛

Reply:
It's a trick to create a triangle using CSS: https://css-tricks.com/snippets/css/css-triangle/

Reply:
Oh wow thank you. I will look at it.